src/modules/blockchainApplication/manage/hooks/useCurrentApplication.test.js

```diff
@@ -39,7 +39,7 @@ describe('useCurrentApplication hook', () => {
     act(() => {
       setCurrentApplication(mockApplications[0]);
     });
-    expect(client.create).toHaveBeenCalledWith({ rpc: expect.stringMatching('wss'), rest: expect.stringMatching('http') });
+    expect(client.create).toHaveBeenCalledWith({ rpc: expect.stringMatching('ws'), rest: expect.stringMatching('http') });
```

Comment:
it should  be wss for now the server dosnt have the ssl

Reply:
@soroushm This test was failing after the change to `ws` in one of the mocks and it needed to be updated. It can be reverted once the server has SSL